include/libtorrent/alert_types.hpp

```diff
@@ -2403,7 +2403,11 @@ namespace libtorrent
 		sha1_hash info_hash;
 
 		int num_peers() const;
-		void peers(std::vector<tcp::endpoint>& peers) const;
+
+#ifndef TORRENT_NO_DEPRECATE
+		void peers(std::vector<tcp::endpoint>& v) const;
```

Comment:
please mark this with the TORRENT_DEPRECATED macro too, to emit warnings when used